components/image/Image.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { SHOW_IMAGES } from 'proton-shared/lib/constants';
+import Button from '../button/Button';
+import useMailSettings from '../../hooks/useMailSettings';
+
+const Image = (props) => {
+    const [{ ShowImages }] = useMailSettings();
+    const [showAnyways, setShowAnyways] = useState(false);
+
+    const handleClick = () => setShowAnyways(true);
+
+    if (ShowImages & SHOW_IMAGES.REMOTE || showAnyways) {
```

Comment:
Before to enter in this condition you need to make sure `src` is defined and is a valid URL (and not base64).